packages/fx-core/src/common/local/projectSettingsHelper.ts

```diff
@@ -26,15 +27,15 @@ export class ProjectSettingsHelper {
       (pluginName) => pluginName === ResourcePlugins.Bot
     );
 
-  public static includeAuth(projectSettings: ProjectSettings | undefined): boolean {
+  public static includeAAD(projectSettings: ProjectSettings | undefined): boolean {
     const selectedPlugins = (projectSettings?.solutionSettings as AzureSolutionSettings)
       ?.activeResourcePlugins;
-    const includeAad = selectedPlugins?.some((pluginName) => pluginName === ResourcePlugins.Aad);
-    const includeSimpleAuth = selectedPlugins?.some(
-      (pluginName) => pluginName === ResourcePlugins.SimpleAuth
-    );
+    return selectedPlugins?.some((pluginName) => pluginName === ResourcePlugins.Aad);
+  }
 
-    return includeAad && (!this.includeFrontend(projectSettings) || includeSimpleAuth);
+  public static includeSimpleAuth(projectSettings: ProjectSettings | undefined): boolean {
+    const includeSimpleAuth = IsSimpleAuthEnabled(projectSettings);
+    return !this.includeFrontend(projectSettings) || includeSimpleAuth;
```

Comment:
not blocking this PR: we can keep current logic, but we'd better have a follow up task on the condition. Is it you or qianhao to follow up on this?

Reply:
What will the final condition be?

Reply:
I think finally we will remove the includeSimpleAuth here, but it will be kept for some time. I prefer to keep this until we abandon since there may be some migration V1 problem if we update the logic here.

Reply:
thank you, as offline discussed, we can keep current logic and wait for removing the code totally to reduce some engineering effort.